### PR TITLE
time: Add README to explain the purpose of the time extension library.

### PR DESCRIPTION
**Pull request by mattytrentini**
Attempts to at least partially assist with the problem raised in #725.

### CODE REVIEW

File: python-stdlib/time/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+employs
+[`asctime`](https://docs.python.org/3/library/logging.html#formatter-objects).
+
+For example:
+
+```python
+logging.Formatter('%(asctime)s | %(name)s | %(levelname)s - %(message)s')
+```
+
+The expecteed output might look like:
```
Comment (projectgus) [inline python-stdlib/time/README.md:33]:
```suggestion
The expected output might look like:
```

**Review by projectgus — APPROVED** [review]
This is a very useful thing to explain, thanks @mattytrentini!

File: python-stdlib/time/README.md (hunk 1 above)
Comment (mattytrentini) [reply python-stdlib/time/README.md]: Thanks @projectgus, fixed!

**Comment by dpgeorge** [comment]
Thanks!

**Pull request merged**